packages/react-components/src/molecules/UserProfilePersonalText.tsx

```diff
@@ -31,19 +31,31 @@ const paragraphStyles = css({
 type UserProfilePersonalTextProps = Pick<
   UserResponse,
   'institution' | 'jobTitle' | 'location' | 'teams' | 'role'
->;
+> & {
+  isOwnProfile?: boolean;
```

Comment:
Didn't we just introduce a context for this?

Reply:
Realized the question may sound snarky, so: I think we should not introduce a prop that is passed through, but directly consume the context, right? Its default value is also `false`

Reply:
It's used on the people listing card and we don't want to display the help text on the listing page assumably

Reply:
But there the default value of the context (`false`) will just come into play so it's fine innit?